torch/csrc/jit/codegen/cuda/scheduler/registry.cpp

```diff
@@ -1114,17 +1192,21 @@ void HeuristicSummary::validate() const {
       TORCH_INTERNAL_ASSERT(
           entry_type_map_.count(EntryType::VECTORIZABLE_INPUTS_AND_OUTPUTS));
       TORCH_INTERNAL_ASSERT(
-          entry_type_map_.count(EntryType::MAPPED_INPUTS_OUTPUTS));
+          entry_type_map_.count(EntryType::BROADCAST_BYTE_MULTIPLES));
       break;
     case ScheduleHeuristic::Reduction:
       TORCH_INTERNAL_ASSERT(entry_type_map_.count(EntryType::REDUCTION_TVS));
       TORCH_INTERNAL_ASSERT(
           entry_type_map_.count(EntryType::VECTORIZABLE_INPUTS_AND_OUTPUTS));
+      TORCH_INTERNAL_ASSERT(
+          entry_type_map_.count(EntryType::UNROLLABLE_INPUTS_AND_OUTPUTS));
       break;
     case ScheduleHeuristic::Normalization:
```

Comment:
nitpick: do we want to rename `ScheduleHeuristic::Normalization` to `ScheduleHeuristic::Persistent` ?